inc/Calculator.h

```diff
@@ -13,4 +13,9 @@
 #include <iostream>
 // Add your function declarations here.
 
+class Calculator {
+    public:
+    std::string hello();
```

Comment:
1- Indent line 18 a bit
2- Need to fill out the brief on line 6
3- Wrap inside of namespaces
4- Missing doxygen comments for each function, only on header files